### PR TITLE
Fix broken signal/slot in Meteor Showers plugin

### LINKED ISSUE #2428: Downloading meteor showers
In version 3.0 of Meteor Showers plugin we have a broken signal/slot in download manager for updates of meteor showers data. As a result we have a invalid status in the GUI and warnings in the log.

To get a warning in the log you can open GUI of the plugin and please press "Update..." button to get the issue in the GUI.

### PR DESCRIPTION
**Pull request by worachate001**
### Description

Fixes #2428

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] This change requires a documentation update

### How Has This Been Tested?

**Test Configuration**:
* Operating system: Ubuntu 20.04
* Graphics Card: Nvidia Geforce RTX

## Checklist:

- [x] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (header file)
- [ ] I have updated the respective chapter in the Stellarium User Guide
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: plugins/MeteorShowers/src/MeteorShowersMgr.hpp [hunk 1]
```diff
@@ -317,10 +322,11 @@ private slots:
 	int m_updateFrequencyHours;
 	QString m_url;
 	QDateTime m_lastUpdate;
-	DownloadStatus m_statusOfLastUpdate;
+	UpdateState updateState;
```
Comment (gzotti) [inline plugins/MeteorShowers/src/MeteorShowersMgr.hpp:325]: This plugin's original developer has followed the habit of naming all _member_ variables with a "m_" prefix. I think you should keep this habit in this plugin for consistency.

File: plugins/MeteorShowers/src/gui/MSConfigDialog.hpp [hunk 2]
```diff
@@ -71,8 +74,9 @@ private slots:
 private:
 	MeteorShowersMgr* m_mgr;
 	Ui_MSConfigDialog* m_ui;
-
+	QTimer* updateTimer;
```
Comment (gzotti) [inline plugins/MeteorShowers/src/gui/MSConfigDialog.hpp:77]: There is a m_updateTimer in the MeteorShowerMgr.hpp. Are they fulfilling different purposes? (I have not built/tested the code yet)

File: plugins/MeteorShowers/src/MeteorShowersMgr.hpp (hunk 1 above)
Comment (worachate001) [reply plugins/MeteorShowers/src/MeteorShowersMgr.hpp]: done.

File: plugins/MeteorShowers/src/gui/MSConfigDialog.hpp (hunk 2 above)
Comment (worachate001) [reply plugins/MeteorShowers/src/gui/MSConfigDialog.hpp]: Thanks for pointing it out. In my understanding, they're unrelated and no need to declare here.

File: plugins/MeteorShowers/src/MeteorShowersMgr.cpp [hunk 3]
```diff
@@ -90,8 +90,8 @@ void MeteorShowersMgr::init()
 
 	// Set up download manager and the update schedule
 	m_networkManager = StelApp::getInstance().getNetworkAccessManager();
-	updateState = CompleteNoUpdates;
-	m_updateTimer = new QTimer(this);
+	m_updateState = CompleteNoUpdates;
+	QTimer* m_updateTimer = new QTimer(this);
```
Comment (alex-w) [inline plugins/MeteorShowers/src/MeteorShowersMgr.cpp:94]: No need `QTimer*` here (see `hpp` file with declaration)

Comment (worachate001) [reply plugins/MeteorShowers/src/MeteorShowersMgr.cpp]: done.

**Comment by alex-w** [comment]
The date of next update is not changed after successful updating

**Review by alex-w — APPROVED** [review]

**Pull request merged**